docroot/themes/humsci/humsci_basic/src/scss/components/_vertical-linked-card.scss

```diff
@@ -238,3 +225,7 @@
     }
   }
 }
+
+.hb-stretch-vertical-linked-cards {
+  height: 100%;
+}
```

Comment:
Just one final change before merging. Please remove this too, is not needed and might cause problems on some cases.